package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "knex-mysql-deadlock",
+  "version": "1.0.0",
+  "description": "A custom knex mysql dialect, which does deadlock retries",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mindflash/knex-mysql-deadlock.git"
+  },
+  "author": "mindflash",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Mindflash/knex-mysql-deadlock/issues"
+  },
+  "homepage": "https://github.com/Mindflash/knex-mysql-deadlock#readme",
+  "dependencies": {
+    "knex": "^0.12.6"
```

Comment:
Yeah, because it supports many dialects, it wouldn't be good for it to forcibly install all of the adapters for you if you don't use them. It only attempts to require the dialect package after you pass in your config specifying which dialect you want knex to use. 🙂